README.md

```diff
@@ -37,15 +48,18 @@ $ curl \
     http://localhost:5000/predictjson
 ```
 
+### Output
+The output is a scalar value: `0` or `1`, which corresponds to whether or not a student is predicted to be successful by the end of the year, as measured by their final grade. A value of `1` means that a student is predicted to be successful whereas `0` means otherwise.
+
 ## Model Description
 
 ### Input Features
 The features we used in training our model are `health`, `absences`, `age`, and `G2`.
 
 ### Improvement from the Baseline Model
-By taking in G2 as an additional feature, our retrained model achieves an f1 score of 0.9861111111111112 on the test set. In comparison, the f1 score of the baseline model on the test set is 0.5185185185185185.
+By taking in G2 as an additional feature, our retrained model achieves an f1 score of `0.986 `on the test set. In comparison, the f1 score of the baseline model on the test set is `0.519`.
```

Comment:
I don't think this metric result was from the test set.

Reply:
The original algorithm had no cross-validation, so it only measured based on the training set

Reply:
Oh. In that case, should we run additional metrics so that the two models are benchmarked against the same dataset? @thisisalbertliang

Reply:
After looking at the two notebooks in more detail, it doesn't seem like we're splitting the datasets at all across test and train. I think that's something we should probably incorporate to make sure that we're not overfitting on the train dataset.

Reply:
I added cross-validation in #9 but it hasn't been merged